libflux/flux-core/src/semantic/bootstrap.rs

```diff
@@ -205,12 +266,22 @@ fn generate_values(
                     continue;
                 }
                 let typ = format!("{}", types[&name].normal());
-                values.push(DocValue {
-                    pkgpath: pkgpath.to_string(),
-                    name: name.clone(),
-                    doc,
-                    typ,
-                });
+                match &types[&name].expr {
+                    MonoType::Fun(_f) => {
+                        // generate function doc
+                        let function = FunctionDoc::new_with_args(name.clone(), doc, typ);
+                        members.insert(name.clone(), Doc::Function(Box::new(function)));
+                    }
+                    _ => {
+                        let builtin = ValueDoc {
+                            name: name.clone(),
+                            headline: doc,
+                            description: None,
+                            flux_type: typ,
+                        };
+                        members.insert(name.clone(), Doc::Value(Box::new(builtin)));
+                    }
```

Comment:
Do we have any builtins that are not functions? If no, then I guess this block would be essentially dead code? But maybe we have non-function builtins, I am really not sure.

Reply:
I don't know if we have any builtins that aren't functions, but it's technically possible, right?

Reply:
Yes, you're right, better to leave it in.